Multisig/UI/Safe Management/Add Safe/EnterSafeAddressViewController/EnterSafeAddressViewController.swift

```diff
@@ -36,6 +38,12 @@ class EnterSafeAddressViewController: UIViewController {
         nextButton.isEnabled = false
 
         navigationItem.rightBarButtonItem = nextButton
+
+        if let address = preselectedAddress {
```

Comment:
Isn't it safer if we did this part in viewDidAppear then we assign preselectedAddress = nil?

Reply:
I prefer to use viewDidAppear over dispatchAfter because of viewController life cycle

Reply:
because of the life cycle, I put it to the viewDidLoad - so that this code would be executed just once